source/platforms/bitbucket_server/BitBucketServerGit.ts

```diff
@@ -131,23 +131,56 @@ const bitBucketServerChangesToGitJSONDSL = (
 
 const bitBucketServerDiffToGitStructuredDiff = (diffs: BitBucketServerDiff[]): GitStructuredDiff => {
   // We need all changed lines with it's type. It will convert hunk segment lines to flatten changed lines.
-  const segmentValues = { ADDED: "add", CONTEXT: "normal", REMOVED: "del" }
+  const segmentValues = { ADDED: "add", CONTEXT: "normal", REMOVED: "del" } as const
   return diffs.map(diff => ({
     from: diff.source && diff.source.toString,
     to: diff.destination && diff.destination.toString,
     chunks:
       diff.hunks &&
       diff.hunks.map(hunk => ({
+        content: `@@ -${hunk.sourceLine},${hunk.sourceSpan} +${hunk.destinationLine},${hunk.destinationSpan} @@`,
```

Comment:
💭 Is that too much? https://www.gnu.org/software/diffutils/manual/html_node/Detailed-Unified.html

Reply:
Nah, that looks right enough to me